subworkflows/local/peaks.nf

```diff
@@ -137,7 +138,7 @@ workflow PEAKS {
     emit:
     affinity_ratio = AFFINITY_RATIO.out.combined
     affinity_sum = AFFINITY_SUM.out.combined
-    enhancers = ROSE.out.enhancers
+    tf_bindings = ch_peaks
```

Comment:
I would prefer if this would be called `candidate_regions` or something like this, because at this point we do not know yet where TFs are binding